news/this-week-in-fluvio-0012.md

```diff
@@ -57,9 +57,11 @@ Error:
    2: WASM module is not a valid 'filter' SmartStream. Are you missing a #[smartstream(filter)] attribute?
 ```
 
+[connectors]: ../docs/connectors
```

Comment:
maybe you have a good reason for doing this... so I'll stop commenting on relative references.